src/cascadia/TerminalConnection/pch.h

```diff
@@ -7,6 +7,9 @@
 
 #pragma once
 
+#define WIN32_LEAN_AND_MEAN
+#define _SILENCE_CXX17_ALLOCATOR_VOID_DEPRECATION_WARNING

```

Comment:
Huh what? Why? At minimum leave the reason why this is necessary.

(We should scope down the suppression to the minimal possible compilation unit, though, if we can.)